modules/op-authenticating-to-an-oci-registry.adoc

```diff
@@ -0,0 +1,49 @@
+// This module is included in the following assembly:
+//
+// *cicd/pipelines/using-tekton-chains-for-pipelines-supply-chain-security.adoc
+
+:_content-type: PROCEDURE
+[id="authenticating-to-an-oci-registry_{context}"]
+= Authenticating to an OCI registry
+
+Before pushing signatures to an OCI registry, cluster administrators must configure Tekton Chains to authenticate with the registry. The Chains controller uses the same service account under which the task runs execute. To set up a service account with the necessary credentials for pushing signatures to an OCI registry, perform the following steps:
+
+[discrete]
+.Procedure
+
+. Set the namespace and name of the Kubernetes service account.
++
+[source,terminal]
+----
+$ export NAMESPACE=<namespace> <1>
+$ export SERVICE_ACCOUNT_NAME=<service_account> <2>
+----
+<1> The namespace associated with the service account.
+<2> The name of the service account.
+
+. Create a `.dockerconfig` type Kubernetes secret.
++
+[source,terminal]
+----
+$ oc create secret docker-registry registry-credentials \
+  --docker-server=<gcr.io> \ <1>
+  --docker-username=<_json_key> \ <2>
```

Comment:
Is the `_` prefix in `_json_key` intentional?

Reply:
Okay. You can disregard. I was asking because it might cause formatting issues if openshift-docs implements the SSG guidance to italicize replaceable values. But we'll address that if it ever comes up.

Reply:
You can mark this as "resolved."